### PR TITLE
Improved logging

### PR DESCRIPTION
**Pull request by bochkver**
Improved logging in -v mode that provides info about where each library provider is connected 
https://github.com/TeskaLabs/asab/issues/254

### CODE REVIEW

File: asab/library/providers/azurestorage.py [hunk 1]
```diff
@@ -93,7 +94,7 @@ async def _load_model(self):
 			)
 
 		self.Model = model
-
+		L.info("AzureStorage library provider {} is connected.".format(self.Path))
```
Comment (ateska) [inline asab/library/providers/azurestorage.py:97]:
```
L.info("Connected.", struct_data={'path':self.Path})
```

**Review by ateska — CHANGES_REQUESTED** [review]
Better to use structured data.

Also @eliska-n was the primary requestor of this, so she could provide more (and faster) feed back.

**Comment by eliska-n** [comment]
yes, when debugging I would like to be ensured that the provider connects where I want. (Where I think my configuration targets.)
In the best scenario I would like to see something like that:
`ZooKeeper library provider connected to: 'zk://servername:2181/some/subpath' `

**Comment by eliska-n** [comment]
Or using the `struct_data`... I don't care. But I think the user/developer/me/whoeverelse would like to see the full URL to be really sure that the provider is looking into the right node. Noone wants to go through the code and seek for the explanation of the `path` or `self.Path`.

**Review by ateska — APPROVED** [review]

**Review by eliska-n — APPROVED** [review]

**Pull request merged**